pkg/apis/managed/v1alpha1/subjectpermission_types.go

```diff
@@ -53,7 +53,7 @@ type Condition struct {
 	// +optional
 	Message string `json:"message,omitempty"`
 	// ClusterRoleName in which this condition is true
-	ClusterRoleName string `json:"clusterRoleName"`
+	ClusterRoleNames []string `json:"clusterRoleName"`
```

Comment:
With this changing to `[]string`, how does `Message` relate? Is the `Message` common to all members of the `[]string`?

Reply:
The only time when we will have multiple clusterRoleNames will be when all clusterRoleBindings and/or roleBindings are created successfully. In both cases, the message is generic "successfully created clusterRoleBindings" and/or "successfully created rolebindings. Otherwise logic stays the same as before.